scripts/test_config.yaml

```diff
@@ -1,54 +1,58 @@
 # config.yaml
+potentials:
+  - provider: "physics-ml"
+    model_name: "MACE"
+    precision: 64
+    position_scaling: 10.0
+    output_scaling: 4.184 * 627
+    model_path: "/guardowl/tests/data/physics-ml/mace"
+
+  - provider: "openmm-ml"
+    model_name: "ANI2x"
+
 tests:
-  - protocol: "alanine_dipeptide_protocol"
+  - protocol: "alanine_dipeptide_test"
     env: "solution"
     ensemble: "npt"
-    nnp: "ani2x"
-    implementation: "torchani"
     annealing: false
     nr_of_simulation_steps: 50
     temperature: 300
 
-  - protocol: "alanine_dipeptide_protocol"
+  - protocol: "alanine_dipeptide_test"
     env: "vacuum"
     ensemble: "npt"
-    nnp: "ani2x"
-    implementation: "torchani"
     annealing: false
     nr_of_simulation_steps: 500
     temperature: 300
 
-  - protocol: "hipen_protocol"
-    hipen_idx: [0, 1, 15]
-    nnp: "ani2x"
-    implementation: "torchani"
+  - protocol: "small_molecule_test"
+    smiles:
+      [
+        r"CCOc1ccc2nc(/N=C\c3ccccc3O)sc2c1",
```

Comment:
No need to keep double quotes in yaml files as this enables special character checking. These can be replaced with single quotes:

```python
'CCOc1ccc2nc(/N=C\c3ccccc3O)sc2c1',
'Cn1cc(Cl)c(/C=N/O)n1',
'S=c1cc(-c2ccc(Cl)cc2)ss1',
```